### PR TITLE
Hyperledger Identus Q3 2024 report

### PR DESCRIPTION
**Pull request by elribonazo**

### CODE REVIEW

File: project-reports/2024/2024-Q4-Hyperledger-Identus.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+nav_exclude: true
+---
+
+Created by Javier Ribó with input from other maintainers
+
+
+# Project Health
+
+The project is still new to the Hyperledger Ecosystem and continues to have steady growth of contributors.
+The project can now be found on the LFX dashboard which can now be checked for health insights. In the last quarter we have seen an increase on the number of github issues created while the active contributors have gone down 8%.
```
Comment (DannieBGoode) [inline project-reports/2024/2024-Q4-Hyperledger-Identus.md]: can you link the [LFX Dashboard](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=identus&routedFrom=Github&bestPractice=false&repository=all&dateFilters=Last%20Quarter&dateRange=2024-07-01%20to%202024-09-30&compare=PP&granularity=week&hideBots=true)

File: project-reports/2024/2024-Q4-Hyperledger-Identus.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+
+# Questions/Issues for the TOC
+
+Is there a way to access funds for open source contributions and development?
```
Comment (DannieBGoode) [inline project-reports/2024/2024-Q4-Hyperledger-Identus.md]: where did this question come from?

**Review by DannieBGoode — COMMENTED** [review]
Left some comments

File: project-reports/2024/2024-Q4-Hyperledger-Identus.md (hunk 1 above)
Comment (elribonazo) [reply project-reports/2024/2024-Q4-Hyperledger-Identus.md]: Link is now added u can solve the conversation!

File: project-reports/2024/2024-Q4-Hyperledger-Identus.md (hunk 2 above)
Comment (elribonazo) [reply project-reports/2024/2024-Q4-Hyperledger-Identus.md]: From the previous report in Identus, removed it anyway its now showing N/A

**Review by DannieBGoode — COMMENTED** [review]
looks good

File: project-reports/2024/2024-Q4-Hyperledger-Identus.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Identus
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q4-Hyperledger-Identus.md:3]:
```suggestion
title: 2024 Q4 Hyperledger Identus
```

With Q4 reports, we are moving to the new TAC governance site. Could you please close this PR and re-open a PR in the https://github.com/lf-decentralized-trust/governance report and add the report to the `tac/project-updates` folder?

File: project-reports/2024/2024-Q4-Hyperledger-Identus.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+
+
+# Maintainer Diversity
+
+We currently have 7 maintainers from which 3 are from outside of IOG, and we are constantly working on onboarding new members to the community by making workshops, community calls and partnering with some projects
+
+# Contributor Diversity
+
+At the moment, the main contributors of source code is IOG at a 75% of the contributions.
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q4-Hyperledger-Identus.md]:
```suggestion
At the moment, the main contributors of source code are from IOG at 75% of the contributions.
```

File: project-reports/2024/2024-Q4-Hyperledger-Identus.md (hunk 3 above)
Comment (DannieBGoode) [reply project-reports/2024/2024-Q4-Hyperledger-Identus.md:3]: I've created this PR  https://github.com/LF-Decentralized-Trust/governance/pull/44/files

File: project-reports/2024/2024-Q4-Hyperledger-Identus.md (hunk 4 above)
Comment (elribonazo) [reply project-reports/2024/2024-Q4-Hyperledger-Identus.md]: pushed, thanks!

**Comment by elribonazo** [comment]
Closing PR in favor of https://github.com/LF-Decentralized-Trust/governance/pull/44/

**Pull request closed without merging**